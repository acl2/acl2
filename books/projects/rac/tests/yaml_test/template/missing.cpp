// RAC begin

int foo() {
  return missing<3>();
}
