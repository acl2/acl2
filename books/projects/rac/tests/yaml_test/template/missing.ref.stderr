missing.cpp:4 (9-16): Unknown symbol `missing`
 4 |   return missing<3>();
   |          ^^^^^^^
